cmd/repository_test.go

```diff
@@ -238,3 +188,16 @@ func piped(t *testing.T) func() string {
 		return strings.TrimSpace(b.String())
 	}
 }
+
+// TEST REPO URI:  Return URI to repo in ./testdata of matching name.
+// Suitable as URI for repository override. returns in form file://
+// Must be called prior to mktemp in tests which changes current
+// working directory as it depends on a relative path.
+// Repo uri:  file://$(pwd)/testdata/repository.git (unix-like)
+//            file: //$(pwd)\testdata\repository.git (windows)
+func testRepoURI(name string, t *testing.T) string {
```

Comment:
I thought I refactored something like this to use HTTP.

Reply:
```go
func TestRepoURI(name string, t *testing.T) string {
	t.Helper()
	wd, err := os.Getwd()
	if err != nil {
		t.Fatal(err)
	}
	gitRoot := filepath.Join(wd, "testdata")
	hostPort := RunGitServer(t, gitRoot)
	return fmt.Sprintf(`http://%s/%s.git`, hostPort, name)
}
```

Reply:
You copy-pasted unfixed version.

Reply:
That was prior to the fix, as this PR has been open for quite some time now...